src/components/Header/ListExport.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { connect } from 'react-redux'
+
+const ListExport = (props) => {
+    return (
+      <a href={url(props.type,props.data)} download={props.download}><p>{props.title}</p></a>
+    )
+  }
+
+function mapDispatchToProps(dispatch) {
+    return {
+
+    }
+  }
+function url(type,data){
+  let blob ={}
+  if(type==="json"){
+     blob = new Blob([JSON.stringify(data)],{type:""})
```

Comment:
```suggestion
     blob = new Blob([JSON.stringify(data, null 2)],{type:""})
```

by doing this, you get JSON with nice formatting :)